packages/server/src/internalPlugin.ts

```diff
@@ -12,6 +12,7 @@ export interface InternalApolloServerPlugin<TContext extends BaseContext>
   // Used to identify a few specific plugins that are instantiated
   // by default if not explicitly used or disabled.
   __internal_plugin_id__(): InternalPluginId;
+  __is_disabled_plugin__(): boolean;
```

Comment:
nbd, and this matches the existing pattern, but is it helpful that this is a function and not just a boolean?

Reply:
I just assumed I had a reason for doing it this way before with the other one :)  Will change both